daemon/driver.go

```diff
@@ -22,13 +22,20 @@ import (
 	protocol2 "gopkg.in/bblfsh/sdk.v2/protocol"
 )
 
+type ServiceV1 interface {
```

Comment:
Please document all the exported members. In this case there's a bunch of existing use, please add comments at least for the new names introduced here (cf. https://github.com/golang/go/wiki/CodeReviewComments#doc-comments).

I noticed a lot of our files lack clear doc comments. I'll file an issue for adding them in a more comprehensive way to the existing code.